sdk/ml/azure-ai-ml/azure/ai/ml/_exception_helper.py

```diff
@@ -15,7 +15,7 @@
     SCHEMA_VALIDATION_ERROR_TEMPLATE,
     YAML_CREATION_ERROR_DESCRIPTION,
 )
-from azure.ai.ml.exceptions import ErrorTarget, ValidationErrorType, ValidationException
+from azure.ai.ml.exceptions import ErrorTarget, ValidationErrorType, ValidationException, MlException
```

Comment:
Does the lower case "l" bother you as much as it does to me?

Reply:
haha it does. If we don't have a reason for it to be lowercase, I can make it MLException